helm/operator/templates/operator-deployment.yaml

```
@@ -40,6 +40,9 @@ spec:
       topologySpreadConstraints:
       {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.operator.priorityClassName }}
+      priorityClassName: {{ . }}
+      {{- end }}
```

For uniformity, I would suggest:
```suggestion
      {{- with .Values.operator.priorityClassName }}
      priorityClassName:
      {{- toYaml . | nindent 8 }}
      {{- end }}
```

---

Your thoughts?

---

yeah!, same I though yesterday, but tried that and crease a carriage return that seems unecesary, ie:
```
priorityClassName:
        prioriryClass
```

The current code outpus as follows, that is more natural.
```
priorityClassName: className
```